documentation/Functions.md

```diff
@@ -322,15 +322,35 @@ downloadFile("https://bit.ly/2BCkFa9", "file", "downloads")
 ***
 
 ### buildNg <a name="buildNg"></a>
+This function is used to build Angular project.
+
 #### parameter 
-1. Path to the angular project, relative to workspace
-2. (Optional) Custom output directory.
+This function consist of two parameter.
+
+1. First parameter:
+    - **Required**
+    - **Type**- String
+    - **Description**-It is the path to the Angular project, relative to workspace.
+
+2. Second parameter:
+    - **optional**
+    - **Type**- String
+    - **Description**- It is the custom output directory.
 #### example 
-buildNg("exampleAngularProject")
-Will build the angular project to default output directory defined in angular.json outputPath key, normally set to dist/.
+* buildNg("exampleAngularProject")
+Will build the Angular project to default output directory defined in angular.json outputPath key, normally set to dist/.
+
+* buildNg("exampleAngularProject", "testOutput")
+Will build the Angular project to output directory testOutput.
+
+Note:
+1. To use only one parameter, you should have Angular CLI installed and the folder should exist. This can be done by using the devonfw-ide or by installing it manually with the npmInstall function.
+
+2. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
+
+3. No background script will run.
 
-buildNg("exampleAngularProject", "testOutput")
-Will build the angular project to output directory testOutput.
+4. To execute this function devonfw-ide must be installed.
```

Comment:
That's not correct. Only the Angular CLI has to be installed, as you mentioned above. This is also possible without devonfw-ide

Reply:
Changed it!!